tools/lint/main_ni.c

```diff
@@ -576,6 +577,8 @@ fill_context(int argc, char *argv[], struct context *c)
             } else if (!strcasecmp(optarg, "lyb")) {
                 c->schema_out_format = 0;
                 c->data_out_format = LYD_LYB;
+            } else if (!strcasecmp(optarg, "feature-param")) {
+                c->feature_param_format = 1;
```

Comment:
What about `c->data_out_format`? All `if` branches set it somehow, but this one doesn't. Is it intentional?

Reply:
I omitted it, because it is already set to 0 (in function main_ni there is `struct context c = {0};`).